server/controllers/socket.controller.ts

```diff
@@ -236,8 +238,8 @@ async function handleOptionSelected(
 }
 
 // TODO: how to handle thrown errors?
-export const handleWs = (socket: WebSocket) => {
-	const userId: string = crypto.randomUUID();
+export const handleWs = (socket: WebSocket, userId: string) => {
+	console.debug(`Handling WS with the following data: userId: ${userId}`);
```

Comment:
Are we handling the transfer of selection/confidence when a player rejoins anywhere? If I make a selection and refresh, do I lose my selection or am I able to keep it? I think that the main value of this ticket is that you can refresh and not lose any state.

Reply:
Not sure, honestly, as the selection cards aren't in yet. Maybe I should hold off on this until we merge that one.

Reply:
That PR just so happens to be ready for review #35 👀